packages/core/src/core/prompts.ts

```diff
@@ -34,228 +34,238 @@ export function getCoreSystemPrompt(userMemory?: string): string {
       throw new Error(`missing system prompt file '${systemMdPath}'`);
     }
   }
+  if (
+    process.env.OPENAI_MODEL?.startsWith('qwen3') &&
+    process.env.OPENAI_BASE_URL?.includes('dashscope')
```

Comment:
Please add a separate function 'UseSystemTemplate()' or sth similar. Move these kind of config to a separate config json file for supported OPENAI_BASE_URL and/or OPENAI_MODEL list. Do not use startswith or includes

Reply:
@wenyuanyu your suggestion is great! I will improve this PR by placing the system prompts settings in the user's configuration file ~/.qwen/settings.json, so that users can configure the system prompt autonomously.